.github/workflows/zeebe-release.yml

```diff
@@ -244,6 +244,50 @@ jobs:
           fi
           shopt -u nocasematch # reset it
           echo "result=${PRE_RELEASE}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.RELEASE_BRANCH }}
+      - name: Install Zeebe changelog tool
+        run: |
+          wget https://github.com/zeebe-io/zeebe-changelog/releases/download/0.7.0/zeebe-changelog_0.7.0_Linux_i386.tar.gz
+          tar -xzvf zeebe-changelog_0.7.0_Linux_i386.tar.gz
+          chmod +x zcl
```

Comment:
💭 There is probably a better way to do this

Reply:
There's actions to do that, but I haven't used any enough to strongly recommend one. That only advantage I can think of is they might do some caching? :shrug: